pkg/config/validation/validation.go

```diff
@@ -95,6 +97,27 @@ var (
 		http.MethodTrace:   true,
 	}
 
+	// golang supported gRPC status: https://github.com/grpc/grpc-go/blob/master/codes/codes.go
+	supportedGRPCStatus = map[string]codes.Code{
```

Comment:
Can we remove this duplication and have these code mapping defined at a common place?

Reply:
any suggestion? cause I cannot find a common place or any `types` dir. And the route.go(pilot/pkg/...) and validate.go(pkg/...) are in very different path.

Reply:
somewhere in  'pkg/util/grpc.go` ?

Reply:
Cool, I will do this change tomorrow, thanks~

Reply:
Done, please take a look.